catalog/opensearch/catalog-opensearch-source/src/test/java/ddf/catalog/source/opensearch/impl/TestOpenSearchSource.java

```diff
@@ -110,6 +114,69 @@
+                        + "            <ns3:metacard xmlns:ns3=\"urn:catalog:metacard\" xmlns:ns2=\"http://www.w3.org/1999/xlink\" xmlns:ns1=\"http://www.opengis.net/gml\"\r\n"
+                        + "                xmlns:ns4=\"http://www.w3.org/2001/SMIL20/\" xmlns:ns5=\"http://www.w3.org/2001/SMIL20/Language\" ns1:id=\"4535c53fc8bc4404a1d32a5ce7a29585\">\r\n"
+                        + "                <ns3:type>ddf.metacard</ns3:type>\r\n"
+                        + "                <ns3:source>ddf.distribution</ns3:source>\r\n"
+                        + "                <ns3:geometry name=\"location\">\r\n"
+                        + "                    <ns3:value>\r\n"
+                        + "                        <ns1:Point>\r\n"
+                        + "                            <ns1:pos>32.8751900768792 13.1874561309814</ns1:pos>\r\n"
+                        + "                        </ns1:Point>\r\n"
+                        + "                    </ns3:value>\r\n"
+                        + "                </ns3:geometry>\r\n"
+                        + "                <ns3:dateTime name=\"created\">\r\n"
+                        + "                    <ns3:value>2013-01-31T16:22:31.648-07:00</ns3:value>\r\n"
+                        + "                </ns3:dateTime>\r\n"
+                        + "                <ns3:dateTime name=\"modified\">\r\n"
+                        + "                    <ns3:value>2013-01-31T16:22:31.648-07:00</ns3:value>\r\n"
+                        + "                </ns3:dateTime>\r\n"
+                        + "                <ns3:stringxml name=\"metadata\">\r\n"
+                        + "                    <ns3:value>\r\n"
+                        + "                        <ns6:xml xmlns:ns6=\"urn:sample:namespace\" xmlns=\"urn:sample:namespace\">Example description.</ns6:xml>\r\n"
+                        + "                    </ns3:value>\r\n"
+                        + "                </ns3:stringxml>\r\n"
+                        + "                <ns3:string name=\"metadata-content-type-version\">\r\n"
+                        + "                    <ns3:value>myVersion</ns3:value>\r\n"
+                        + "                </ns3:string>\r\n"
+                        + "                <ns3:string name=\"metadata-content-type\">\r\n"
+                        + "                    <ns3:value>myType</ns3:value>\r\n"
+                        + "                </ns3:string>\r\n"
+                        + "                <ns3:string name=\"title\">\r\n"
+                        + "                    <ns3:value>Example title</ns3:value>\r\n"
+                        + "                </ns3:string>\r\n" + "            </ns3:metacard>\r\n"
+                        + "        </Resource:Resource>\r\n" + "    </entry>\r\n" + "</feed>";
```

Comment:
Ah, my bad.  I just kept it this way to keep the class consistent.  If there is a strong opinion I can extract the ~4 resources in there